package.json

```diff
@@ -9,7 +9,6 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@splidejs/vue-splide": "^0.3.5",
     "core-js": "^3.6.5",
```

Comment:
Bummer that we didn't get to use the carousel, but at least you know how to use it for your next project! ;)

Reply:
Yup, I thought the same thing.